CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## [Unreleased]
 
 ### Release highlights
+* Support for adding a `change-id` Git header behind the configuration setting `git.change-id`. If
+  this is enabled, `jj` will attempt to preserve the Change ID of a commit across Git remotes by
+  storing it in a Git `change-id` header. e.g. If you `jj git push` to a remote, edit it, then
+  `jj git fetch` your remote work, it should preserve the logical Change ID through those
+  modifications. This feature currently defaults to "false".
```

Comment:
Add "We plan to make this feature unconditionally enabled in a few releases." or something like that